ssrando.spec

```
@@ -49,6 +49,14 @@ else:
 with open('version-with-git.txt','w') as f:
   f.write(VERSION)
 
+hiddenimports = []
+
+try:
+  import certifi
+  hiddenimports.append('certifi')
+except:
```

Empty excepts look bad, why not have
```suggestion
except ModuleNotFondError:
```

---

It doesn't do any error handling anyways, so I don't see the benefit of adding which error gets thrown here. Plus I don't have to worry about if there is any other error that can possibly be thrown

---

Catching all errors (if you don't re-raise) is usually bad practice; you don't want to assume all errors should be ignored.
In this case, imagine for some reason importing certifi blocks (eg. infinite loop)
Then, if the user interrupts, the rando resumes in the `except` case, while the user just wanted to completely interrupt everything